R/build-home-citation.R

```diff
@@ -31,39 +31,55 @@ read_citation <- function(path = ".") {
 data_home_sidebar_citation <- function(pkg = ".") {
   pkg <- as_pkgdown(pkg)
 
-  if (!has_citation(pkg$src_path)) {
-    return(character())
-  }
   sidebar_section(
     heading = "Citation",
-    bullets = a(sprintf(tr_("Citing %s"), pkg$package), "authors.html")
+    bullets = a(sprintf(tr_("Citing %s"), pkg$package), "authors.html#citation")
```

Comment:
that ID wouldn't be translated, or it could, maybe, but it'd add complexity.

Reply:
Agreed; it's not user facing so I think it's fine as is.